frontend/apps/reader/modules/readerstyletweak.lua

```diff
@@ -727,7 +727,7 @@ function ReaderStyleTweak:onToggleStyleTweak(tweak_id, item, no_notification)
         else
             self.doc_tweaks[tweak_id] = nil
         end
-        text = _("Disabled style tweak: ")
+        text = T(_("Off: %1"), self.tweaks_in_dispatcher[tweak_id])
```

Comment:
I guess that for such short and standard On/Off:, we should add some translation context and/or a @ translator comment.

Reply:
Can we just make these notifications always in two lines? First line is “Style tweak enabled/disabled:” in any language, second line is the css name.

Reply:
Currently notification is a TextWidget
https://github.com/koreader/koreader/blob/b1c90260e40dc5b1f6c58dc9cac9b26786572015/frontend/ui/widget/textwidget.lua#L2

Reply:
Yes, I have thought about having it multilines in the past, but gave up. We don't really need it to be more than one line, we're mostly fine elsewhere - and most people don't care about notifications, so the smaller the better :)

Reply:
I do think it's a good suggestion all around. Concatenating `.. "\n" ..` (or maybe two) bothers translators the minimum amount and the display would likely be better too. Except for that single line notification aspect. :-)

Reply:
But yes, also notifications should be as small as possible, forgot about that for a second.